frontend/tests/dangerZone.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { Page, expect } from "@playwright/test";
+import { test, realms } from "./util/common";
+import { USERS, login } from "./util/user";
+
+for (const realmType of realms) {
```

Comment:
Would be nice of the `realms` constant would have a better name, which clearly explains what it is. Also, stuff like `realmType === "User" ? "morgan" : "sabine"` are repeated in a bunch of places. Maybe both things can be fixed in one go. Maybe `realms` can contain actual objects with `type: "User" | "..."` but also `userWithModeratorAccess` and `name` or sth. But still renaming it to mhhh. `testRealms`?

Mh but might not be too easy, as in different tests, different realms are used. Maybe just rename it to `realmTypes`?